WordPress/Classes/ViewRelated/Suggestions/SuggestionsTableView.m

```diff
@@ -38,12 +40,36 @@ - (instancetype)initWithSiteID:(NSNumber *)siteID
     return self;
 }
 
+- (void)setUseTransparentHeader:(BOOL)useTransparentHeader
+{
+    _useTransparentHeader = useTransparentHeader;
+    [self updateHeaderStyles];
+}
+
+#pragma mark Private methods
+
+- (void)updateHeaderStyles
+{
+    if (_useTransparentHeader) {
+        [self.headerView setBackgroundColor: [UIColor clearColor]];
+    } else {
+        [self.headerView setBackgroundColor: [UIColor colorWithRed:0.f green:0.f blue:0.f alpha:0.3f]];
+    }
+    
+}
+
 - (void)setupHeaderView
 {
     _headerView = [[UIView alloc] init];
-    _headerView.backgroundColor = [UIColor colorWithRed:0.f green:0.f blue:0.f alpha:0.3f];
     [_headerView setTranslatesAutoresizingMaskIntoConstraints:NO];
     [self addSubview:_headerView];
+    
+    _separatorView = [[UIView alloc] init];
+    [_separatorView setTranslatesAutoresizingMaskIntoConstraints:NO];
+    [_separatorView setBackgroundColor: [UIColor colorWithRed:0.f green:0.f blue:0.f alpha:0.1f]];
```

Comment:
Yep!  Fixed in https://github.com/wordpress-mobile/WordPress-iOS/commit/60cced5001d63b20a352b1e1f94d9ebdd5891d19